src/less/mixins/utility/utility-mixins.less

```diff
@@ -31,6 +31,11 @@
     @{property}: var(~"--@{token1}", @@token1) var(~"--@{token2}", @@token2);
 }
 
+.customProperty(@property, @token1, @token2, @token3) {
+    @{property}: var(~"--@{token1}", @@token1) var(~"--@{token2}", @@token2)
+        var(~"--@{token3}", @@token3);
```

Comment:
we don't need to add more customProperty mixins now that we don't support IE. Let's not use these and just use CSS custom properties

Reply:
We will continue using them for a little while longer. The mixin provides a nice shorthand syntax too.